pyHS100/pyHS100.py

```diff
@@ -291,8 +291,22 @@ def location(self):
         :rtype: dict
         """
         info = self.sys_info
-        return {"latitude": info["latitude"],
-                "longitude": info["longitude"]}
+        loc = {"latitude": 0.0,
+               "longitude": 0.0}
+
+        if "latitude" in info:
+            loc["latitude"] = info["latitude"]
+
```

Comment:
well if you're at 0.0/0.0, you're literally in the middle of the ocean, which I highly doubt is a plausible case, but it could be useful as a visual cue between set vs not set.